README.md

```diff
@@ -77,3 +79,60 @@ Spørsmål knyttet til koden eller prosjektet kan rettes mot:
 ### For NAV-ansatte
 
 Interne henvendelser kan sendes via Slack i kanalen #team-dagpenger-dev.
+
+## Vanlig oppstart av prosjektet (når alt er satt opp)
+
+1. Start opp Naisdevice (den må bli grønn)
+2. `$ git pull`
+3. `$ nvm use`
+4. `$ npm install`
+5. Bruk [trydgebrukeren](https://nav-it.slack.com/archives/C04RRM5QEAV/p1682666310232779) for å logge inn på de neste tre stegene
+6. Legg inn token fra [dp-rapportering](https://wonderwalled-azure.intern.dev.nav.no/api/obo?aud=dev-gcp.teamdagpenger.dp-rapportering) i `.env`-filen
+7. Legg inn token fra [dp-behandling](https://wonderwalled-azure.intern.dev.nav.no/api/obo?aud=dev-gcp.teamdagpenger.dp-behandling) i `.env`-filen
+8. [dp-vedtak](https://wonderwalled-azure.intern.dev.nav.no/api/obo?aud=dev-gcp.teamdagpenger.dp-vedtak)
```

Comment:
Burde det stå "Legg inn token fra [dp-vedtak](https://wonderwalled-azure.intern.dev.nav.no/api/obo?aud=dev-gcp.teamdagpenger.dp-vedtak) i `.env`-filen" her? 

Slik som på de to forrige linjene?